src/edu/wright/cs/raiderplanner/controller/MainController.java

```diff
@@ -123,6 +123,8 @@ public static void initialise() {
 				MainController.setSpc(study);
 				plannerFile = MainController.ui.savePlannerFileDialog();
 				loadFile(plannerFile);
+				/*This is cating a general exception because the
```

Comment:
There should be a space between '*' and 'This'. Also, 'cating' should be 'catching'. There should also be a period at the end of the sentence and the final '*/' should be on its own line.